### PR TITLE
Remove openmp as it's not used

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

**Comment by calum-chamberlain** [comment]
openmp is used in the OSX and Linux style builds, but not in the Windows builds.  What was the motivation for all these changes?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,25 +12,22 @@ source:
...
+  number: 1
 
 requirements:
   build:
     - python
     - setuptools
     - toolchain
     - future
-    - numpy x.x
+    - numpy 1.11
```
Comment (calum-chamberlain) [inline recipe/meta.yaml]: Looks like this should be 1.11.* as per [docs](https://conda-forge.org/docs/meta.html#building-against-numpy)

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,25 +12,22 @@ source:
...
     - fftw
-    - openmp
   run:
     - python
     - setuptools
     - future
-    - numpy x.x
+    - numpy >=1.11
     - fftw
-    - openmp
```
Comment (calum-chamberlain) [inline recipe/meta.yaml:33]: OSX and Linux compile C routines using openmp. But, you might be right, compilation may work without (if this is a standard C library included with gcc?)

File: ci_support/run_docker_build.sh [hunk 3]
```diff
@@ -57,69 +58,9 @@ conda clean --lock
...
-    conda build /recipe_root --quiet || exit 1
-    upload_or_check_non_existence /recipe_root conda-forge --channel=main || exit 1
-
-    set -x
-    export CONDA_NPY=113
-    export CONDA_PY=36
-    set +x
-    conda build /recipe_root --quiet || exit 1
-    upload_or_check_non_existence /recipe_root conda-forge --channel=main || exit 1
+conda build /recipe_root --quiet || exit 1
```
Comment (calum-chamberlain) [inline ci_support/run_docker_build.sh:61]: Why have you removed separate build cases here for different Python versions?

Comment (calum-chamberlain) [reply ci_support/run_docker_build.sh:61]: Handled by `config.yml`, which then calls this script, okay.

**Review by calum-chamberlain — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (isuruf) [reply recipe/meta.yaml:33]: openmp conda package is the llvm one. You may want to add libgcc to run requirements for gcc one

File: ci_support/run_docker_build.sh (hunk 3 above)
Comment (isuruf) [reply ci_support/run_docker_build.sh:61]: These changes were added after rerendering with conda-smithy

File: recipe/meta.yaml (hunk 2 above)
Comment (calum-chamberlain) [reply recipe/meta.yaml:33]: Circle seems happy without it (and the tests that run the openMP version pass), so it looks good. Thanks.

File: ci_support/run_docker_build.sh (hunk 3 above)
Comment (calum-chamberlain) [reply ci_support/run_docker_build.sh:61]: 👍

**Review by calum-chamberlain — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (calum-chamberlain) [reply recipe/meta.yaml:33]: Unfortunately it looks like OSX builds need the openmp package (as you might expect with it being LLVM).  Is there a way to just include it for those builds?

**Review by calum-chamberlain — CHANGES_REQUESTED** [review]
OSX builds fail without openmp.

**Pull request merged**